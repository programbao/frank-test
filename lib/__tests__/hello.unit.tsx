// 测试函数
function add(a: number, b: number) {
  return a + b;
}
describe("我的第一个测试用例", () => {
  it("函数输出等于目标", () => {
    expect(add(1, 2)).toEqual(3);
  });
});
