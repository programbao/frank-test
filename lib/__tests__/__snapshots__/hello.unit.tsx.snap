// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  按钮
</div>
`;

exports[`我的第一个测试用例 是一个div 1`] = `
<div>
  按钮
</div>
`;
